### ISSUE TITLE
mythweb max_input_vars bug and fix

### ISSUE THREAD

**Issue by Steve-Goodey** [MythTV/mythtv#307]
* **Platform**: Linux mythbuntu 5.4.0-65-generic #73-Ubuntu SMP Mon Jan 18 17:25:17 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
* **MythTV version**: fixes/31 (v31.0+fixes.202101081714.016630a35c-ubuntu20.04.1)
* **Package version**: mythbuntu-ubuntu-31-focal.list
* **Component**: mythbackend mythweb

### What steps will reproduce the bug?

Mythweb, Settings, Channel Info. Selecting commfree on some channels and going to end of page and pressing save.

### How often does it reproduce? Is there a required condition?

Reproduce everytime.

### What is the expected behaviour?

comfree setting should stay enabled after pressing save.

### What do you see instead?

Screen refreshes and shows the page but with an error message.
"Warning: Unknown: Input variables exceeded 1000. To increase the limit change max_input_vars in php.ini. in Unknown on line 0"
"The fix is insert php_value max_input_vars 10000 in /etc/apache2/sites-available/mythweb.conf in the section with the other php config settings."
This is from https://forum.mythtv.org/viewtopic.php?f=6&t=3788&p=18206&hilit=max_input_vars#p18206 by JeffDwork.

**Comment by garybuhrmaster** [comment]
It really feels like this issue should have been opened against the MythTV/mythweb repo (not the MythTV/mythtv repo).

**Comment by Steve-Goodey** [comment]
Yes, sorry about that. That's what happens when you let an idiot raise an issue. 

Should I close this and raise it in the correct place?
